Added Inputs

atharvabhanage02 opened the pull request:
Added Text , Number , Password , Success , Warning , Error , Radio and Checkbox Input Fields
Output is in the README file

atharvabhanage02 committed:
Added Inputs

Added Text , Number , Password , Success , Warning , Error , Radio and Checkbox Input Fields

JantuDeb commented on components/components.css:
```diff
@@ -55,4 +55,10 @@
+     /* Inputs */
+    --input-font-size:1rem;
+    --text-grey:#808080;
+    --input-width:300px;
```
Try to avoid using fixed width for input, you may need it in different places where you require different width.

atharvabhanage02 replied:
Yes , got it . I'll make the changes
Thanks for correcting me

JantuDeb reviewed (approved):
Suggested some changes.

sks-21 reviewed (approved):
The code looks nice. I would suggest you to prefer relative units over absolute units.

t007rushi commented on components/inputs/inputs.css:
```diff
@@ -0,0 +1,30 @@
+    font-size: 1.25rem;
+}
+.radio-input,.checkbox-input{
+    height: 15px;
```
Use relative units like rem

atharvabhanage02 replied:
Yes, I'll do that . Thanks for suggesting that .

atharvabhanage02 committed:
Added Inputs

Removed the hardcoded width and used relative units to increase the maintain consistency throughout the Project
